README.md

```diff
@@ -88,6 +88,15 @@ include a reset of your choice.
 Also, if you want to shave ~150ms from each test run, you can remove the require
 from the template. In that case, you'll have to require it yourself.
 
+You can also customise the tests, failure, and details window heights by setting
+the following custom vars to the desired value:
+
+- `kaocha-runner-tests-win-min-height`
+
+- `kaocha-runner-failure-win-min-height`
+
+- `kaocha-runner-details-win-max-height`
+
```

Comment:
I think adding the docstring to each point here would be useful, since these are hardly self explanatory.

Reply:
Done